### PR TITLE
Hide SMTP config password and do not change the set password if blank…

### PR DESCRIPTION
**Pull request by norkans7**
… is submitted

### CODE REVIEW

**Comment by rowanseymour** [comment]
Did this come from a ticket?

**Comment by norkans7** [comment]
Yes @rowanseymour

File: temba/orgs/views.py [hunk 1]
```diff
@@ -865,6 +867,11 @@ def clean(self):
                     smtp_password = self.cleaned_data.get('smtp_password', None)
                     smtp_port = self.cleaned_data.get('smtp_port', None)
 
+                    config = self.instance.config_json()
+                    existing_username = config.get(SMTP_USERNAME, '')
+                    if not smtp_password and existing_username == smtp_username:
```
Comment (rowanseymour) [inline temba/orgs/views.py]: User could change the username and end up saving a blank password, no?

Comment (norkans7) [inline temba/orgs/views.py]: we have a validation down for empty password

Comment (norkans7) [inline temba/orgs/views.py]: and a unit test for that as well

Comment (rowanseymour) [inline temba/orgs/views.py]: 👍

**Pull request merged**